lib/config/options/index.ts

```diff
@@ -3067,6 +3067,46 @@ const options: RenovateOptions[] = [
     default: 90,
     globalOnly: true,
   },
+  {
+    name: 'dockerMaxPages',
+    description:
+      'By default, Renovate fetches up to 20 pages of Docker tags from registries, but this maximum can be adjusted by setting this option.',
+    type: 'integer',
+    default: 20,
+    globalOnly: true,
+  },
+  {
+    name: 'gitlabAutoMergeableCheckAttempts',
+    description:
+      'If set to a positive integer, Renovate tries this many times to check if a merge request on GitLab is mergeable, before trying to automerge.',
```

Comment:
Umm, no -ve integers won't be caught during validation. However I don't think anyone will use a negative integer as delay, cause it could cause time to move faster :)

Reply:
Good idea to validate though

Reply:
On it: https://github.com/renovatebot/renovate/pull/29178